src/kernels/jupyter/jupyterUriProviderWrapper.ts

```diff
@@ -14,6 +14,9 @@ export class JupyterUriProviderWrapper implements IJupyterUriProvider {
         return this.provider.id;
     }
     public getQuickPickEntryItems(): vscode.QuickPickItem[] {
+        if (!this.provider.getQuickPickEntryItems) {
```

Comment:
Extensions such as github or others won't need to implement `getQuickPickEntryItems` (see type definition)